File: Dummy/hawkshaw/library/datalayer/room/telephony/CallRecordEntity.java
package com.hawkshaw.library.datalayer.room.telephony;

import Y1.K;

public final class CallRecordEntity {
    private int id;
    private final String name;
    private final String number;

    public CallRecordEntity(String str, String str2) {
        K.n(str, "name");
        K.n(str2, "number");
        this.name = str;
        this.number = str2;
    }

    public final int getId() {
        return this.id;
    }

    public final String getName() {
        return this.name;
    }

    public final String getNumber() {
        return this.number;
    }

    public final void setId(int i5) {
        this.id = i5;
    }
}

File: app/src/main/java/com/hawkshaw/library/datalayer/room/telephony/CallRecordEntity.kt
package com.hawkshaw.library.datalayer.room.telephony

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * Entity representing a phone number to be recorded
 */
@Entity(tableName = "call_record_entity")
data class CallRecordEntity(
    @ColumnInfo(name = "name")
    var name: String = "",
    
    @ColumnInfo(name = "number")
    var number: String = "",

    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "id")
    var id: Int = 0
) 